src/frontend/handlers/contracts.ts

```diff
@@ -1,8 +1,8 @@
-import { RequestHandler } from "express";
 import { getContracts } from "../config";
+import { PageParamsGetter } from "../pages";
 
-export const contractsHandler: RequestHandler = async (_, response) => {
-  response.render("contracts.njk", {
+export const contractsParamsGetter: PageParamsGetter<any> = async (_) => {
```

Comment:
why the explicit `any`?

Reply:
It complains without it.  Maybe I should specify it explicitly?

Reply:
Oh, right, it should just be {} here I think.